2022/22xxx/CVE-2022-22183.json

```diff
@@ -1,18 +1,165 @@
 {
```

Comment:
Invalid JSON file (invalid state)

Reply:
Sorry for missing that.  "STATE": "PUBLIC"